src/Server.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace React\Http;
+
+use React\Http\Middleware\LimitConcurrentRequestsMiddleware;
+use React\Http\Middleware\RequestBodyBufferMiddleware;
+use React\Http\Middleware\RequestBodyParserMiddleware;
+use React\Socket\ServerInterface;
+
+/**
+ * Facade around StreamingServer with sane defaults for 80% of the use cases.
+ * The requests passed to your callable are fully compatible with PSR-7 because
+ * the body of the requests are fully buffered and parsed, unlike StreamingServer
+ * where the body is a raw ReactPHP stream.
+ *
+ * Wraps StreamingServer with the following middleware:
+ * - LimitConcurrentRequestsMiddleware
+ * - RequestBodyBufferMiddleware
+ * - RequestBodyParserMiddleware
+ * - The callable you in passed as first constructor argument
+ *
+ * All middleware use their default configuration, which can be controlled with
+ * the the following configuration directives from php.ini:
+ * - upload_max_filesize
+ * - post_max_size
+ * - max_input_nesting_level
+ * - max_input_vars
```

Comment:
This also respects `file_uploads` and `max_file_uploads` ini settings as documented for `RequestBodyParserMiddleware`.

Reply:
Ah my bad, added them :+1: